f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
I believe I only ran the two armed variant.  My test environment was set up for that and I probably didn't run the 1 armed test.


Sent from my iPad

> On Jun 22, 2016, at 9:29 PM, Matt Greene <notifications@github.com> wrote:
> 
> In f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py:
> 
> >          for bigip in bigips:
> > +            self.service_adapter.get_vlan(vip, bigip, network_id)
> Alright. I'm ok with this for now provided the traffic test passes for both 1-armed and 2-armed deployments. Did you already run both variations?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>